sqreen/manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+    "integration_id": "sqreen",
+    "name": "sqreen",
+    "manifest_version": "0.0.1",
+    "guid": "318a1d52-85ec-47ef-8174-72017881b360",
+    "maintainer": "support@sqreen.com",
```

Comment:
let's update this to help@datadoghq.com

Reply:
@anshumgargdd how will questions sent to this email be redirected to our team?